Turnstile-Ontology/02-Software/03-Implementation/.output.o.rack

```diff
@@ -0,0 +1,12 @@
+:- multifile build_with/5, build_from/2, build_inputs/2, build_outputs/2,
```

Comment:
These `.rack` files are generated data that shouldn't be committed.

Reply:
removed .rack file